Content.Server/LandMines/LandMineSystem.cs

```diff
@@ -14,14 +15,38 @@ public sealed class LandMineSystem : EntitySystem
 
     public override void Initialize()
     {
+        SubscribeLocalEvent<LandMineComponent, GetVerbsEvent<AlternativeVerb>>(OnGetAltVerbs);
         SubscribeLocalEvent<LandMineComponent, StepTriggeredOnEvent>(HandleStepOnTriggered);
         SubscribeLocalEvent<LandMineComponent, StepTriggeredOffEvent>(HandleStepOffTriggered);
 
         SubscribeLocalEvent<LandMineComponent, StepTriggerAttemptEvent>(HandleStepTriggerAttempt);
     }
 
+    /// <summary>
+    /// Adds a verb to prime the landmine to activate if somebody steps on it.
+    /// </summary>
+    private void OnGetAltVerbs(EntityUid uid, LandMineComponent component, GetVerbsEvent<AlternativeVerb> args)
```

Comment:
If at all possible try to put this in shared so it isn't laggy! You might need to do some funky function overloading but it shouldn't be too hard. If you want help / need more examples let me know or ask in the discord!

Some examples:
#32420
#33622
#32496

Reply:
I am struggling a bit with understanding exactly the nuances of the implementation of this. I've updated the relevant code with what I *believe* is the right pattern.